public/static/docs/command-reference/metrics/diff.md

```diff
@@ -0,0 +1,94 @@
+# metrics diff
+
+Show a table of changes between
+[metrics](/doc/command-reference/metrics#description) among
+<abbr>repository</abbr> versions.
+
+## Synopsis
+
+```usage
+usage: dvc metrics diff [-h] [-q | -v]
+                        [--targets [TARGETS [TARGETS ...]]]
+                        [-t TYPE] [-x XPATH] [-R] [--show-json]
+                        [a_ref] [b_ref]
+
+positional arguments:
+  a_ref     Git reference from which the diff begins. If omitted,
+           `HEAD` (latest commit) is used.
+  b_ref     Git reference until which the diff ends. If omitted,
+            the current workspace is used instead.
```

Comment:
@efiop can we call these "revisions" instead? I.e. `a_rev`: Git revision from... Or maybe "commits". Larger explanation in https://github.com/iterative/dvc/pull/3244#pullrequestreview-348990111.

Reply:
@jorgeorpinel @efiop so, what is the progress with this rename in DVC core?

Reply:
iterative/dvc/issues/3245